packages/headless/src/modules/manufacturer/services/manufacturer.service.ts

```diff
@@ -255,4 +258,38 @@ export class ManufacturerService {
       ) as DeleteManufacturerResponse;
     }
   }
+
+  /**
+   * The ManufacturesCount function returns the number of Manufactures
+   * If the number of Manufactures is equal to zero then returns the empty message
+   * @returns { Promise<Object> } Object of Success or Error
+   */
+  async manufacturersCount(): Promise<ManufacturersCountResponse> {
+    const count = await this.manufacturerRepo.getManufacturersCount();
+
+    if (count === 0) {
+      return this.helper.serviceResponse.successResponse(
+        {
+          count: 0,
+          message: GetManufacturersCountSuccessMessages.MANUFACTURER_IS_EMPTY,
+        },
+        HttpStatus.OK,
+      ) as ManufacturersCountResponse;
+    } else if (count > 0) {
+      return this.helper.serviceResponse.successResponse(
+        {
+          count,
+          message:
+            GetManufacturersCountSuccessMessages.MANUFACTURERS_COUNT_LOADED_SUCCESSFULLY,
+        },
+        HttpStatus.OK,
+      ) as ManufacturersCountResponse;
```

Comment:
lets discuss next day. why we need an api to show the count only

Reply:
I am going to close this PR because the all manufacturers count is found already when calling the getAllManufacturers api except passing query as skip and limit